### PR TITLE
Review subgraphs

### PR DESCRIPTION
**Pull request by ajsantander**
* Removes connect-thegraph-template
Maintenance overhead was too high. It's just a copy of connect-thegraph-voting. A template package in the future should be an automated tool. We can revive the package when that happens. For now, I moved all docs on how to create TheGraph connectors to docs/thegraph-connectors.md.
* Minor modifications to tokens and voting templates.
Mainly all core templates have access to all ABIs, which was why some of our subgraphs were failing to sync.
* Cli demos use official subgraphs.

### CODE REVIEW

File: docs/thegraph-connectors.md [hunk 1]
```diff
@@ -1,26 +1,54 @@
-# TheGraph template for Aragon subgraphs
+# TheGraph connectors
+
+This section contains information about how our TheGraph connectors work, and how to use one of the existing connectors as a template for your own Aragon app connector.
+
+All existing connectors target all instances of a specific app, and can be deployed on any network. However, they can be modified to connect to any other Aragon app. In theory, it can even be used to connect to anything Aragon related, not just apps.
+
+## Architecture
+
+The connectors are composed of a root project which implements the base TheGraph connector, and an underlaying subgraph. Both components are specified below.
```
Comment (Evalir) [inline docs/thegraph-connectors.md:9]:
```suggestion
The connectors are composed of a root project which implements the base TheGraph connector, and an underlying subgraph. Both components are specified below.
```

File: docs/thegraph-connectors.md [hunk 2]
```diff
@@ -1,26 +1,54 @@
...
+
+The "src" folder contains the TheGraph connector wrapper which simply uses "@aragon/connect-thegraph" to perform GraphQL queries, parse them, and present them in the form of entity objects consistent with the "@aragon/connect" API.
+
+Use "index.ts" to pick which entities and objects are exposed to other packages.
+
+In "connect.ts", add functions that the entities of your connector will use. For example, the Vote entity will call its "castsForVote(...): Promise<Cast[]>" function. These functions all follow the same structure of (1) performing a query, (2) parsing the results of a query, and (3) wrapping and returning the results in the appropriate entity.
+
+Each of the steps described above is separated in the "entities", "parsers", and "queries" folders for clarity.
+
+Notice the file "queries/types.ts". This file is generated by a 3rd party tool, and provides Typescript types for GraphQL results. It is completely optional, but can be super handy when writing parsers.
```
Comment (Evalir) [inline docs/thegraph-connectors.md:21]: I think we should specify which tool is creating this `types.ts` file.

File: docs/thegraph-connectors.md [hunk 3]
```diff
@@ -1,26 +1,54 @@
...
+
+Queries are defined using "graphql-tag", which allows using fragments. Fragments are useful for when your queries start getting complicated and you want to reuse chunks of code in them.
```
Comment (Evalir) [inline docs/thegraph-connectors.md:23]: Should we link to the official graphql docs for [fragments](https://graphql.org/learn/queries/#fragments)?

File: docs/thegraph-connectors.md [hunk 4]
```diff
@@ -34,47 +62,53 @@ If you'd like to disable a specific injected data source, such as for example th
...
-When a reducer is run, it's run in the context of the data source that defined it. For example, hooks are triggered by src/base/Kernel.ts when the NewAppProxy event is detected in an Organization. You need to include the missing ABI in manifest/templates/Kernel.template.yaml for it to be available in this reducer.
+When a reducer is run, it's run in the context of the data source that defined it. For example, hooks are triggered by src/base/Kernel.ts when the NewAppProxy event is detected in an Organization. You need to include the missing ABI in manifest/templates/contracts/Kernel.template.yaml for it to be available in this reducer. As a rule of thumb, if your data source will be triggered by a base Aragon data source (Organization templates, Organization factories, etc), include your its ABI in the manifest/templates/contracts files.
 
 * My callHandlers aren't working
 Unfortunately, TheGraph does not support callHandlers in Rinkeby. For this reason, this template avoids them altoghether. In general, we prefer to code subgraphs in a way that is compatible with all networks. Alternatively, if you by all means need to use this feature, consider hosting your own subgraph.
```
Comment (Evalir) [inline docs/thegraph-connectors.md:114]:
```suggestion
Unfortunately, TheGraph does not support callHandlers in Rinkeby. For this reason, this template avoids them altogether. In general, we prefer to code subgraphs in a way that is compatible with all networks. Alternatively, if you by all means need to use this feature, consider hosting your own subgraph.
```

File: docs/thegraph-connectors.md (hunk 2 above)
Comment (ajsantander) [reply docs/thegraph-connectors.md:21]: This should be changed soon. I'm working in a way to automate things. For now, there's comments in the types files.

File: docs/thegraph-connectors.md (hunk 3 above)
Comment (ajsantander) [reply docs/thegraph-connectors.md:23]: @macor161 it would be great if after our first release you could review this doc and include this suggestion

**Pull request merged**